benchmarks/synth-bm/src/native_transfer.rs

```diff
@@ -84,6 +85,11 @@ pub async fn benchmark(args: &BenchmarkArgs) -> anyhow::Result<()> {
         rpc_response_handler.handle_all_responses().await;
     });
 
+    let transaction_stat_service =
```

Comment:
I have a request: enable (or disable) `transaction_stat_service` through a command line flag.

The reason is that in my current workloads I don't always need this functionality. I could ignore the output, but I think it's cleaner to have a flag to turn this off

Reply:
Done.